packages/wasm/crate/src/planner.rs

```diff
@@ -384,6 +388,54 @@ pub async fn plan_transaction(
         return Err(anyhow!("PositionWithdraw not yet implemented").into());
     }
 
+    for tpr::ActionDutchAuctionSchedule { description } in request.dutch_auction_schedule_actions {
+        let description = description
+            .ok_or_else(|| anyhow!("missing description in Dutch auction schedule action"))?;
+        let input: Value = description
+            .input
+            .ok_or_else(|| anyhow!("missing input in Dutch auction schedule action"))?
+            .try_into()?;
+        let output_id: Id = description
+            .output_id
+            .ok_or_else(|| anyhow!("missing output ID in Dutch auction schedule action"))?
+            .try_into()?;
+        let min_output: Amount = description
+            .min_output
+            .ok_or_else(|| anyhow!("missing min output in Dutch auction schedule action"))?
+            .try_into()?;
+        let max_output: Amount = description
+            .max_output
+            .ok_or_else(|| anyhow!("missing max output in Dutch auction schedule action"))?
+            .try_into()?;
+
+        actions.push(ActionPlan::ActionDutchAuctionSchedule(
+            ActionDutchAuctionSchedule {
+                description: DutchAuctionDescription {
+                    start_height: description.start_height,
+                    end_height: description.end_height,
+                    step_count: description.step_count,
+                    input,
+                    output_id,
+                    min_output,
+                    max_output,
+                    nonce: [0; 32], // TODO: Use a real nonce
```

Comment:
I think this needs to be filled out right?

Reply:
Ah, scratch that — I just read the very next line 🤦🏻‍♂️ will fix, one sec...

Reply:
@erwanor [fixed](https://github.com/penumbra-zone/web/pull/1017/commits/d111fea61c0b59cbe2e7bb8c7be1d7a13ac92e0a)

Reply:
The reason we can't set the nonce to zero is that doing so would restrict the ability to submit an auction with those parameters to only the first person who tries (that would be funny actually)